market_ssua/urls.py

```diff
@@ -1,5 +1,5 @@
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
-
+    path("user", include("users.urls"))
```

Comment:
URI 첫번째 계층은 복수로 만들어주세요~
```suggestion
    path("users", include("users.urls"))
```

Reply:
넵 알겠습니다 😁